test.py

```diff
@@ -114,6 +114,18 @@ async def test_send_req():
     print(f"send_req time: {format(execution_time, '.9f')}ms")
 
 
+async def test_send_req_arrow():
```

Comment:
it would be nice to utilize the returned `res` value for something to test if it really maps to pyarrow properly

Reply:
What behavior do we want in this case?

Reply:
done